pipelines/broad/arrays/validate_chip/ValidateChip.dev.options.json

```diff
@@ -0,0 +1,9 @@
+{
+  "read_from_cache": true,
```

Comment:
Do we still need the 3 different options files?

Reply:
I'm not sure.  Can't remember what google_project is doing there anyways.  I'll see what happens.

Reply:
Looks like they're not needed.  Have updated.

Reply:
👍